src/server/utils/processManager.js

```diff
@@ -145,7 +145,6 @@ export default class ProcessManager {
         rej();
       }
       this.killing = true;
-      const isWin = /^win/.test(process.platform);
       if (!isWin) {
```

Comment:
I think this needs to be:
```suggestion
      if (!isWin()) {
```